plugins/permission-backend-module-opa-wrapper/src/opa-client/opaClient.ts

```diff
@@ -32,22 +32,22 @@ export class OpaClient {
   /**
    * Evaluates a policy against a given input.
    * @param input - The input to evaluate the policy against.
-   * @param opaPackage - The OPA package to use. You can optionally provide the package here, otherwise it will be taken from the app-config.
+   * @param opaEntryPoint - The entry point into the OPA policy to use. You can optionally provide the entry point here, otherwise it will be taken from the app-config.
    */
   async evaluatePolicy(
     input: PolicyEvaluationInput,
-    opaPackage?: string,
+    opaEntryPoint?: string,
   ): Promise<PolicyEvaluationResult> {
-    const setOpaPackage = (opaPackage ?? this.opaPackage)?.replace(/\./g, '/');
+    const setEntryPoint = opaEntryPoint ?? this.opaEntryPoint;
     const opaBaseUrl = this.opaBaseUrl;
-    const url = `${opaBaseUrl}/v1/data/${setOpaPackage}`;
+    const url = `${opaBaseUrl}/v1/data/${setEntryPoint}`;
 
     if (!opaBaseUrl) {
       this.logger.error('The OPA URL is not set in the app-config!');
       throw new Error('The OPA URL is not set in the app-config!');
     }
 
-    if (!setOpaPackage) {
+    if (!setEntryPoint) {
       this.logger.error(
         'The OPA package is not set in the evaluatePolicy method or in the app-config!',
```

Comment:
Should say entrypoint here rather than package I guess

Reply:
Ah yes! ty!